docs/architecture/psa-migration/md-cipher-dispatch.md

```diff
@@ -190,8 +197,7 @@ Note: PSA cipher is built on Cipher, but PSA AEAD directly calls the underlying
 
 Here are some reasons why calling `psa_xxx()` to perform a hash or cipher calculation might not be desirable in some circumstances, explaining why the application would arrange to call the legacy software implementation instead.
 
-* `MBEDTLS_PSA_CRYPTO_C` is disabled.
-* There is a PSA driver which has not been initialized (this happens in `psa_crypto_init()`).
+* For ciphers `MBEDTLS_PSA_CRYPTO_CLIENT` is disabled.
```

Comment:
This isn't specific to ciphers. If `MBEDTLS_PSA_CRYPTO_CLIENT` is disabled, then there are no `psa_xxx` functions in the build, so we can't call them.

Also, the part about drivers is still relevant.

```suggestion
* `MBEDTLS_PSA_CRYPTO_CLIENT` is disabled.
* There is a PSA driver which has not been initialized (this happens in `psa_crypto_init()`).
```